package.json

```diff
@@ -5,14 +5,13 @@
   "main": "5m5v-bot.js",
   "dependencies": {
     "js-yaml": "^3.13.1",
-    "twit": "^2.2.11"
+    "rettiwt-api": "^2.4.2"
   },
   "devDependencies": {
-    "mocha": "^6.2.0"
+    "nodeunit": "^0.11.3"
```

Comment:
Switch back to `nodeunit` because all our tests were set up for it and I didn't have time to update them.